RPLCD/lcd.py

```diff
@@ -182,13 +189,18 @@ def __init__(self, pin_rs=15, pin_rw=18, pin_e=16, pins_data=[21, 22, 23, 24],
         self.pins = PinConfig(rs=pin_rs, rw=pin_rw, e=pin_e,
                               d0=block1[0], d1=block1[1], d2=block1[2], d3=block1[3],
                               d4=block2[0], d5=block2[1], d6=block2[2], d7=block2[3],
+                              backlight=pin_backlight,
                               mode=numbering_mode)
+        self.backlight_active_low=backlight_active_low
         self.lcd = LCDConfig(rows=rows, cols=cols, dotsize=dotsize)
 
         # Setup GPIO
         GPIO.setmode(self.numbering_mode)
         for pin in list(filter(None, self.pins))[:-1]:
             GPIO.setup(pin, GPIO.OUT)
+        if pin_backlight != None:
```

Comment:
When comparing with `None`, the `is` operator should be used: `if pin_backlight is not None:`